cmd/web.go

```diff
@@ -242,6 +242,16 @@ func runWeb(ctx *cli.Context) {
 					m.Patch("/hooks/:id:int", bind(api.EditHookOption{}), v1.EditRepoHook)
 					m.Get("/raw/*", middleware.RepoRef(), v1.GetRepoRawFile)
 					m.Get("/archive/*", v1.GetRepoArchive)
+					m.Post("/forks", v1.ForkRepo)
+
+					m.Get("/commits/:commitId", middleware.RepoRef(), v1.CommitById)
+					m.Get("/commits/current", middleware.RepoRef(), v1.CurrentCommit)
```

Comment:
This is not consistent, I would prefer call a API for get HEAD commit of a ref, and use returns SHA to call  "/commits/:commitId".